src/Grid.ts

```diff
@@ -299,12 +299,34 @@ abstract class Grid<Options extends GridOptions = GridOptions> extends Component
     const mounted: GridItem[] = updated.filter((item) => item.mountState !== MOUNT_STATE.MOUNTED);
     const moreUpdated: GridItem[] = [];
 
+    mounted.filter((item) => {
+      if (!item.hasTransition) {
+        const element = item.element!;
+        const transitionDuration = parseFloat(getComputedStyle(element).transitionDuration);
+
+        if (transitionDuration > 0) {
+          item.hasTransition = true;
+          item.transitionDuration = element.style.transitionDuration;
+          return true;
+        }
+      }
+      return false;
+    }).forEach((item) => {
+      item.element!.style.transitionDuration = "0s";
```

Comment:
I already checked that there is an element in the filter, do I need to put `!` again?

Reply:
Unfortunately, typescript doesn't even handle this.